components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -130,6 +132,8 @@ class APIProviderImpl extends AbstractAPIManager implements APIProvider {
     protected GatewayArtifactsMgtDAO gatewayArtifactsMgtDAO;
     private RecommendationEnvironment recommendationEnvironment;
     private GlobalMediationPolicyImpl globalMediationPolicyImpl;
+    private static final String ENDPOINT_CONFIG_SEARCH_TYPE_PREFIX  = "endpointConfig:";
```

Comment:
Let's move this `private static final` variable *after* the following line: 
```
private ServiceCatalogDAO serviceCatalogDAO = ServiceCatalogDAO.getInstance();
```

Reply:
fixed https://github.com/wso2/carbon-apimgt/pull/11739